manifests/cf-manifest/manifest/010-cf-jobs.yml

```diff
@@ -222,6 +224,9 @@ jobs:
       - name: cf
     update:
       serial: true
+    properties:
+      bbs:
+        url: (( grab meta.bbs.api_location  ))
```

Comment:
btw., don't look at the monitor, look at the metric. Datadog still has this bug with no_data on monitors

Also, you can try deleting the monitor and creating it again by running cf deploy again, which would run TF task to create monitors